Formula/postgresql.rb

```diff
@@ -94,11 +94,13 @@ def install
   end
 
   def post_install
+    require "open3"
     (var/"log").mkpath
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
     end
+    _stdout, _stderr, _status = Open3.capture3("createdb")
```

Comment:
It would be good to verify if it's necessary. I'd suggest perhaps just putting this in the `unless` above?